plugins/nf-amazon/src/main/nextflow/cloud/aws/batch/AwsBatchFileCopyStrategy.groovy

```diff
@@ -122,7 +122,7 @@ class AwsBatchFileCopyStrategy extends SimpleFileCopyStrategy {
             uploads=()
             IFS=\$'\\n'
             for name in \$(eval "ls -1d ${escape.join(' ')}" | sort | uniq); do
-                uploads+=("nxf_s3_upload '\$name' s3:/${Escape.path(targetDir)}")
+                uploads+=("nxf_s3_upload \"\$name\" s3:/${Escape.path(targetDir)}")
```

Comment:
I think these cases aren't actually broken, when the script is preparing the `uploads` command. Since the `'$name'` is part of an inner double quoted string `"nxf_s3_upload ..."`, I think it is parsed correctly.

Reply:
Reverted this change I as confirmed through testing that it is not needed.

Reply:
cool